specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/examples/PolicyStates_GetMetadata.json

```diff
@@ -0,0 +1,14 @@
+{
+  "parameters": {
```

Comment:
We retired the metadata API, after discussing with the ARM team. I think better to remove it from this version.